src/Data/Nat/Coprimality.agda

```diff
@@ -21,7 +21,9 @@ open import Function.Base using (_∘_)
 open import Level using (0ℓ)
 open import Relation.Binary.PropositionalEquality.Core as P
   using (_≡_; _≢_; refl; trans; cong; subst)
-open import Relation.Nullary as Nullary using (¬_; contradiction; yes ; no)
+import Relation.Nullary.Decidable as Dec
+open import Relation.Nullary.Negation using (¬_; contradiction)
+import Relation.Nullary as Nullary
```

Comment:
After the refactoring in v2.0, it should be possible to export all of this via `Relation.Nullary`. I'd only do this fine grained importing if you can eliminate the `Relation.Nullary` imports altogether?

Reply:
OK... IIRC I'd abandoned #2055 partly over such questions. I agree that importing from top-level `Relation.Nullary` might be more robust, but I think I'd been trying to get my head round the precise dependencies (plus using qualified imports in order to do so; eg `Dec.map` (primed or otherwise) seems more informative than finding a bare occurrence of `map`?)

Reply:
As for importing `Relation.Nullary` itself, that partly rests on unresolved issues about where `Recomputable` should live as a definition, together with its inherited pointwise versions in `Unary` and `Binary`. But happy to postpone such niceties, against later refactoring.

Reply:
Done.